### ISSUE TITLE
Error: attempt to store non-zero value in section '.bss' (console.s)

### ISSUE THREAD

**Issue by 4llan** [fabianishere/boot2flappy#1]
Hi there! I'm getting these errors while trying to build:
```/root/boot2flappy/src/efi-x86_64/console.s: Assembler messages:
/root/boot2flappy/src/efi-x86_64/console.s:32: Error: attempt to store non-zero value in section `.bss'
/root/boot2flappy/src/efi-x86_64/console.s:33: Error: attempt to store non-zero value in section `.bss'
/root/boot2flappy/src/efi-x86_64/console.s:34: Error: attempt to store non-zero value in section `.bss'
```
-- Related? https://github.com/Uzebox/uzebox/pull/65

**Comment by fabianishere** [comment]
Hi, thanks for the report! Looks like it might indeed related to the Uzebox issue.

**Comment by fabianishere** [comment]
I have just fixed it in 1ba986f. It should now build on the latest Ubuntu images.
